### DISCUSSION TITLE
Supabase auth with create t3 turbo

### DISCUSSION THREAD

**Discussion by alexander-densley** [t3-oss/create-t3-turbo#1055] [Q&A]
anyone here successfully implement supabase auth across nextjs and expo app with trpc? As it stands in our nextjs app we are doing server side auth, and that uses cookies, so now when the expo app calls the trpc api, the getUser function provided by supabase server client uses cookies and the expo app doesn't have access to those. any thoughts? would love to see what people have done resolve this issue. Thanks!

**Comment by trevorpfiz** [comment]
@alexander-densley take a look at my PR here https://github.com/supabase-community/create-t3-turbo/pull/11
